rfc6266_parser.py

```diff
@@ -168,97 +156,36 @@ def ensure_charset(text, encoding):
         return text
 
 
-def parse_headers(content_disposition, location=None, relaxed=False):
+def parse_headers(content_disposition, location=None):
     """Build a ContentDisposition from header values.
+    :type content_disposition: str|unicode|None
```

Comment:
`build_header` returns bytes, while `parse_headers` receives text. This is asymmetrical.